psi4/src/psi4/libmints/wavefunction.cc

```diff
@@ -85,6 +85,70 @@ Wavefunction::Wavefunction(std::shared_ptr<Molecule> molecule, std::shared_ptr<B
     common_init();
 }
 
+Wavefunction::Wavefunction(std::shared_ptr<Molecule> molecule, std::shared_ptr<BasisSet> basisset, 
+                           std::map<std::string, std::shared_ptr<Matrix>> matrices,
+                           std::map<std::string, std::shared_ptr<Vector>> vectors,
+                           std::map<std::string, Dimension> dimensions, std::map<std::string, int> ints, 
+                           std::map<std::string, std::string> strings, std::map<std::string, bool> booleans, 
+                           std::map<std::string, float> floats)
+    : options_(Process::environment.options),
```

Comment:
Might want to add a point here that it would be good to change this in the future when we can pass around option objects rather than globals.

Reply:
Will do